### PR TITLE
QPPSF-11471: Create 2023 benchmarks file

### PR DESCRIPTION
**Pull request by ckawell-sb**
- New PY2023 Benchmarks File added

- Scripts updated to handle new CSV format

### CODE REVIEW

File: benchmarks/2023/benchmarks-schema.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+$id: https://github.com/CMSgov/qpp-measures-data/versions/0.0.1/benchmarks-schema.yaml
```
Comment (ckawell-sb) [inline benchmarks/2023/benchmarks-schema.yaml:1]: This Schema file is a copy-paste from PY22. Is this what it should be?

**Review by ljackson-sb — APPROVED** [review]

File: scripts/benchmarks/parse-benchmarks-data.js [hunk 2]
```diff
@@ -86,7 +99,7 @@ if (benchmarkYear && performanceYear) {
   process.stdin.on('end', function() {
     // Quote option update to handle 2020 data
     const columns = benchmarkType === 'MCC' ? MCC_BENCHMARK_CSV_COLUMNS : BENCHMARK_CSV_COLUMNS;
-    parse(benchmarksData, {columns, from: 3, quote: '"'}, function(err, records) {
+    parse(benchmarksData, {columns, from: 2, quote: '"'}, function(err, records) {
```
Comment (ljackson-sb) [inline scripts/benchmarks/parse-benchmarks-data.js:102]: what is the reason for this change?

Comment (ckawell-sb) [reply scripts/benchmarks/parse-benchmarks-data.js:102]: I'm guessing in earlier years there were two lines of headers. But with this CSV, the data starts on row 2. I was seeing the first row being ignored until I changed that.

**Review by ljackson-sb — DISMISSED** [review]

File: benchmarks/2023/benchmarks-schema.yaml (hunk 1 above)
Comment (pkasarski) [reply benchmarks/2023/benchmarks-schema.yaml:1]: Yes, updates had been added there.

**Review by pkasarski — APPROVED** [review]

**Review by ljackson-sb — APPROVED** [review]
